Sources/SotoCore/AWSClient.swift

```diff
@@ -455,13 +459,34 @@ extension AWSClient {
         execute: @escaping (AWSHTTPRequest, EventLoop, Logger) -> EventLoopFuture<AWSHTTPResponse>,
         processResponse: @escaping (AWSHTTPResponse) throws -> Output,
         config: AWSServiceConfig,
-        logger: Logger = AWSClient.loggingDisabled,
+        context: LoggingContext,
```

Comment:
You've removed the default argument here so someone has to supply a context, but I guess this is a moot point as eventually we will be getting this from the task local storage

Reply:
Yes, I purposefully broke the API here because I don't intend/recommend someone else to use it. With `TaskLocal`s landing soon we'll likely remove `LoggingContext` entirely, as the feature of automatically logging metadata derived from `Baggage` could live in `Logging` itself.